loc/static/js/main.js

```diff
@@ -46,10 +46,10 @@ jQuery(function($){
         $(this).unbind('submit').submit();
     });
 
-    // init datepickers
-    var start_year = $("#id_date1").val();
-    var end_year = $("#id_date2").val();
-
+    var max_year = $("#id_date2 option:last-child").val();
```

Comment:
Definitely a judgement call on style but JavaScript tends to use camelCase for variable names but I'm not sure it's worth updating the entire file (or e.g. running it through prettier).

Reply:
I ran it through prettier but I did not change the `_` to camelCase.